index.d.ts

```diff
@@ -187,6 +192,14 @@ declare namespace RedisSMQ {
 		 */
 		message: string;
 
+		/**
+		 * Key for the message to deduplicate unique messages
+		 *
+		 * @type {string}
+		 * @memberof SendMessageOptions
+		 */
+		qkey?: string;
```

Comment:
Any reason for labelling the key `qkey` the `q` just makes it more confusing?

Reply:
Consistency with the `qname` option is the reason!